src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/CaseManagementForCaseWorkerService.java

```diff
@@ -185,7 +186,7 @@ public void dateToCurrentPosition(CaseData caseData) {
     }
 
     public void setEt3ResponseDueDate(CaseData caseData) {
-        if (!isNullOrEmpty(caseData.getClaimServedDate()) && isNullOrEmpty(caseData.getEt3DueDate())) {
```

Comment:
is this removal intentional? I think this logic was put in place so that et3DueDate would only get set if a value wasn't present. Removing this means that the value will potentially be updated every single time